test/extension/numeric/test_fixture.hpp

```diff
@@ -27,4 +28,54 @@ auto create_kernel(std::initializer_list<T> const& values)
     return kernel;
 }
 
+template <typename SrcView, typename DstView>
+void row_conv1D_offset_img_generator(SrcView src_view, DstView dst_view, int const offset,
```

Comment:
Good to add comments here about what this function does, else we are gonna have confusion in future the way we did with boundary options.

Reply:
I have added comments, please take a look at them and state whether this comment is resolvable.